### PR TITLE
docs(samples): update samples to include new parent levels (folder and project)

### PR DESCRIPTION
**Pull request by Sita04**
Updating snippets to reflect new parent levels.

### CODE REVIEW

File: samples/snippets/snippets_notification_configs.py [hunk 1]
```diff
@@ -28,11 +28,15 @@ def create_notification_config(organization_id, notification_config_id, pubsub_t
     # TODO: pubsub_topic = "projects/{your-project-id}/topics/{your-topic-ic}"
     # Ensure this ServiceAccount has the "pubsub.topics.setIamPolicy" permission on the new topic.
 
-    org_name = "organizations/{org_id}".format(org_id=organization_id)
+    # parent_name must be in one of the following formats:
```
Comment (owenhuyn) [inline samples/snippets/snippets_notification_configs.py:31]: The function name parameter should change from `organization_id` -> `parent` or `parent_id`.

Comment (Sita04) [reply samples/snippets/snippets_notification_configs.py]: Modified it. Thanks.

Comment (owenhuyn) [reply samples/snippets/snippets_notification_configs.py]: Sorry, my previous comment wasn't exactly clear. What I meant was that the function parameter eg)
`def create_notification_config(organization_id, notification_config_id, pubsub_topic):`

This should be changed to 
`def create_notification_config(parent_id, notification_config_id, pubsub_topic):`

Then line 35 `parent_id = f"organizations/{organization_id}"` should just be removed as the caller should pass in `parent_id` directly.

Let me know if there needs to be more clarification.

**Review by owenhuyn — APPROVED** [review]
LGTM!

**Review by parthea — APPROVED** [review]

**Pull request merged**